src/context/WalletProvider/NewWalletViews/sections/InstalledWalletsSection.tsx

```diff
@@ -0,0 +1,117 @@
+import { Box, Button, Flex, Image, Stack, Text as CText } from '@chakra-ui/react'
+import { uniqBy } from 'lodash'
+import type { EIP6963ProviderDetail } from 'mipd'
+import { useCallback, useMemo } from 'react'
+import { isMobile } from 'react-device-detect'
+import { Text } from 'components/Text'
+import type { KeyManager } from 'context/WalletProvider/KeyManager'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { isMobile as isMobileApp } from 'lib/globals'
+import { useMipdProviders } from 'lib/mipd'
+
+const MipdProviderSelectItem = ({
+  provider,
+  connect,
+  isSelected,
+  isDisabled,
+}: {
+  provider: EIP6963ProviderDetail
+  connect: (adapter: string) => void
+  isSelected: boolean
+  isDisabled: boolean
+}) => {
+  const handleConnect = useCallback(
+    () => connect(provider.info.rdns),
+    [connect, provider.info.rdns],
+  )
+
+  return (
+    <Box
+      as={Button}
+      key={provider.info.rdns}
+      variant='ghost'
+      px={2}
+      ml={-2}
```

Comment:
```suggestion
      px={4}
      ml={-4}
```

Reply:
<img width="303" alt="image" src="https://github.com/user-attachments/assets/45cb5638-d755-4ced-b616-ab0d949c9c88" />

So we give it a bit more space

Reply:
I like space 
<img width="383" alt="image" src="https://github.com/user-attachments/assets/84e1226c-e187-4127-af73-05d3afda20c4" />